### ISSUE TITLE
Parsec app doesn't show up

### ISSUE THREAD

**Issue by yume-chan** [lwouis/alt-tab-macos#206]
![image](https://user-images.githubusercontent.com/1330321/78318937-95fbb000-7598-11ea-8b2b-1559d127060e.png)

I'm using AltTab 3.3.2 on macOS 10.15.3.

the main window of Parsec app (https://parsecgaming.com/) doesn't show up in AltTab switcher.

I also encountered #204 so the Parsec window is already in the same screen with the switcher.

**Comment by lwouis** [comment]
For #204, I'll be releasing a fix in a few minutes

**Comment by lwouis** [comment]
I've downloaded Parsec and reproduced the issue. Interestingly, it only happens when AltTab is open *then* you open Parsec. If you open AltTab *after* Parsec is already open, it shows properly.

This indicates an issue with subscribing to the Accessibility events "new window opened" from the Parsec app. This happened in the past in #182. There I had to hardcode a specific fix for Octave.app since their launch behavior was bogus.

I wonder what's happening here with Parsec. Hopefully I get more time to investigate soon

**Comment by lwouis** [comment]
Parsec has the same issue as Octave.app: the app starts with `activationPolicy` == `.prohibited`, then later switches to `.regular`. This is not a normal behavior. I will hardcode an exception for Parsec, but I would consider it a bug in Parsec to advertise itself as a background process, then after around 700ms, as a regular app.

For posterity: I played with a workaround that involves retrying for around 10s to check is the `activationPolicy` changes. I thought this may help workaround future apps that have this bug. However I decided to remove it because it means we have even more stuff going in the background and eating CPU. At the end of the day only 2 apps are known to have this defect so far, so easier to whitelist them than to add a general logic that has performance/complexity consequences.
